editor/src/components/canvas/commands/queue-group-true-up-command.ts

```diff
@@ -29,3 +33,18 @@ export const runQueueGroupTrueUp: CommandFunction<QueueGroupTrueUp> = (
     ],
   }
 }
+
+// If the target is in a group, then this will add a command for including the siblings in a trueing up.
+export function getRequiredGroupTrueUps(
+  metadata: ElementInstanceMetadataMap,
+  pathTrees: ElementPathTrees,
+  target: ElementPath,
+): Array<QueueGroupTrueUp> {
+  const parentPath = EP.parentPath(target)
+  if (treatElementAsGroupLike(metadata, pathTrees, parentPath)) {
+    const siblings = MetadataUtils.getSiblingsOrdered(metadata, pathTrees, target)
+    return siblings.map((sibling) => queueGroupTrueUp(sibling.elementPath))
```

Comment:
since we already have access to the siblings here, we could instead create a `pushIntendedBoundsAndUpdateGroups` command, with the intendedBounds array containing all non-reparented siblings and their globalFrames. 
queueGroupTrueUp uses trueUpGroupsForElementAfterDomWalkerRunsResult which has the performance penalty,
pushIntendedBoundsAndUpdateGroups is "free" because it uses the frames provided to the command and does not need to wait for a dom-walker pass.